components/page-header/page-header.component.ts

```diff
@@ -45,7 +47,11 @@ const NZ_CONFIG_MODULE_NAME: NzConfigKey = 'pageHeader';
         <!--back-->
         @if (nzBackIcon !== null) {
           <div (click)="onBack()" class="ant-page-header-back">
-            <div role="button" tabindex="0" class="ant-page-header-back-button">
+            <div
+              role="button"
+              tabindex="0"
+              [class]="!enableBackButton ? 'ant-page-header-back-button__disabled' : 'ant-page-header-back-button'"
```

Comment:
I'd like to hide the button when it's not available, instead of showing the disable style, since it doesn't exist in antd, WDYT?